plugins/transformer/basic/client/spec/spec.go

```diff
@@ -7,10 +7,11 @@ import (
 )
 
 const (
-	KindRemoveColumns    = "remove_columns"
-	KindAddColumn        = "add_column"
-	KindObfuscateColumns = "obfuscate_columns"
-	KindChangeTableNames = "change_table_names"
+	KindRemoveColumns      = "remove_columns"
+	KindAddColumn          = "add_column"
```

Comment:
I'll let @marianogappa comment as well, but I'd rather add support for placeholders in `add_column` like `{{NOW}}` or something like that, which is similar to other places we inject dynamic values (e.g. https://hub.cloudquery.io/plugins/destination/cloudquery/s3/latest/docs#overview-s3-spec)

Reply:
While I really like that idea, `add_column` only adds in a string value while we want this to be a timestamp column

Reply:
We could support configuring a type and try to convert the value to it

Reply:
Or if the value equals `{{NOW}}` use a timestamp type

Reply:
We can go as deep as we want and I'm not really the one to answer how deep. That's the real question to answer here, and I think @bbernays implicitly answered it: a `now()` column is valuable, and _also_ interpolating `now()` in strings doesn't add much value (that's what I infer).

We'd have to document how that works, and add edge cases to the code. Happy to work on it if it's worth it, though.

Reply:
I would prefer to avoid overloading the transformer types at this time, while we are still learning about how users want to use the functionality